cgmml/data_utils/dataset_generation/train_test_split.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+import random
+
+import dbutils
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--db_connection_file', default="dbconnection.json", help='json file with secrets to connect to database')  # noqa: E501
+
+    args = parser.parse_args()
+    db_connection_file = args.db_connection_file
+
+    connector = dbutils.connect_to_main_database(db_connection_file)
+
+    select_not_grouped_person = """
```

Comment:
I was slightly confused by the wording `grouped` (because it reminds me of the GROUP BY which isn't used here)
```suggestion
    select_uncategorized_persons = """
```